Cooldown/ContentView.swift

```diff
@@ -31,7 +31,7 @@ struct ContentView: View {
     }
     
     func isLPMActive() -> Bool {
-        guard let response = run("pmset -g | grep lowpowermode") else {
+        guard let response = run("pmset -g | grep PowerMode") else {
```

Comment:
<img width="465" alt="image" src="https://github.com/user-attachments/assets/693da8e2-8399-4156-a733-971ee838f5cf">

you should use `grep -E "(lowpowermode|PowerMode)"` for backwards compatibility with macOS versions older than Sequoia